mifosng-android/src/main/java/com/mifos/mifosxdroid/online/loanrepaymentschedule/LoanRepaymentSchedulePresenter.java

```diff
@@ -32,19 +32,19 @@ public void attachView(LoanRepaymentScheduleMvpView mvpView) {
     @Override
     public void detachView() {
         super.detachView();
-        if (mSubscription != null) mSubscription.unsubscribe();
+        if (compositeDisposable != null) compositeDisposable.clear();
     }
 
     public void loadLoanRepaySchedule(int loanId) {
         checkViewAttached();
         getMvpView().showProgressbar(true);
-        if (mSubscription != null) mSubscription.unsubscribe();
-        mSubscription = mDataManager.getLoanRepaySchedule(loanId)
+        if (compositeDisposable != null) compositeDisposable.clear();
+        compositeDisposable = mDataManager.getLoanRepaySchedule(loanId)
                 .observeOn(AndroidSchedulers.mainThread())
                 .subscribeOn(Schedulers.io())
-                .subscribe(new Subscriber<LoanWithAssociations>() {
+                .subscribeWith(new DisposableObserver<LoanWithAssociations>() {
```

Comment:
@mohak1712 I already tried that, but those gives me error here. I don't know why `DisposableObserver` gives error. Not only here but same at many places.

Reply:
@Rish02Sharma can u tell what is the error that you are getting